### ISSUE TITLE
main: Remove hardcoded Clear Containers references

### ISSUE THREAD

**Issue by sboeuf** [clearcontainers/runtime#921]
Every reference to Clear Containers should be configured depending on the configuration used. In case we test Kata, we want the logger name to be `kata-runtime` instead of `cc-runtime`. Same thing for references to `Clear Containers`.
Refer to:
https://github.com/clearcontainers/runtime/blob/master/main.go#L33-L34

**Comment by jodh-intel** [comment]
Agreed. The most flexible approach would be to define them in the `Makefile`:

```
PROJECT_NAME = Intel® Clear Containers
PROJECT_BIN_NAME = cc-runtime

USER_VARS += PROJECT_NAME
USER_VARS += PROJECT_BIN_NAME
```

And then add the appropriate code to `GENERATED_CODE` in `Makefile` so that `config-generated.go` contains these values, rather than defining them in `main.go`.
